transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/TransportImpl.java

```diff
@@ -36,8 +37,8 @@
   }
 
   @Override
-  public void send(Event<T> event) {
```

Comment:
We don't want to remove this method to preserve backwards compatibility. pls add a new method, maybe `schedule()` or `sendAsync()`

Reply:
PTAL